File: Worktime/Worktime/ViewModel/PostPassage.cs
namespace Worktime.ViewModel
{
    public class PostPassage
    {
        public int PointerId {  get; set; }

        public string Tag { get; set; } = string.Empty;

        public string LogTime { get; set; } = string.Empty;
        public float Longitude { get; set; } = 0;
        public float Latitude { get; set; } = 0;
    }
}

File: Worktime-mobile/app/src/main/java/b2n/worktime/dto/PostPassage.java
package b2n.worktime.dto;

public class PostPassage {
    public int pointerId;
    public String tag;

    public String logTime;
    public double longitude;
    public double latitude;
    public PostPassage(int _pointerId, String _tag, String _logTime, double _longitude, double _latitude)
    {
        pointerId = _pointerId;
        tag = _tag;
        logTime = _logTime;
        longitude = _longitude;
        latitude = _latitude;
    }
}
